sim/deathknight/army_of_the_dead.go

```diff
@@ -29,6 +30,7 @@ func (dk *Deathknight) registerArmyOfTheDeadCD() {
 		return
 	}
 
+	// TODO: this might be handled already in core code.. double check (if not we might want to enable it)
```

Comment:
It was not. I also taught that if a spell is channeled this will happen automatically so 👍 on adding it